src/config/env.zod.ts

```diff
@@ -88,11 +89,11 @@ export const featureTogglesEnvSchema = z.object({
 export const secretEnvSchema = z.object({
   SYMMETRIC_ENCRYPTION_KEY: z
     .string()
-    .base64({
```

Comment:
question: why do you remove the base64 zod validator? I find it simpler to understand